src/sdl2/video.rs

```diff
@@ -1276,6 +1289,13 @@ impl WindowBuilder {
         self.create_metal_view = true;
         self
     }
+
+    /// Sets shaped state.
```

Comment:
Please change to "Sets shaped state, to create via SDL_CreateShapedWindow instead of SDL_CreateWindow".

Reply:
Fixed!